reporter/otlp_reporter.go

```diff
@@ -70,6 +70,7 @@ type traceAndMetaKey struct {
 	apmServiceName string
 	// containerID is annotated based on PID information
 	containerID string
+	pid         string
```

Comment:
If `addProfileAttributes()` gets updated to also support `AnyValue_IntValue`, `pid` could be of type `int64`. As `pid` is usually a number, we should report it as such and avoid `string`.